node/src/cli.rs

```diff
@@ -49,7 +49,7 @@ pub struct ExportGenesisStateCommand {
 
 	/// Id of the parachain this state is for.
 	///
-	/// Default: 1001
+	/// Default: 1000
 	#[structopt(long)]
```

Comment:
```suggestion
	/// Default: 1000
	#[structopt(default_value = "1000", long)]
```

Reply:
We could add this change or just handle it as part of the command.
https://github.com/paritytech/statemint/pull/82/files#diff-8ce03a5a2b79f3021ff2d6322d13448db8cfd5fb9267517ab4b96d0d299ed45fR99

Reply:
Sure we can keep the `unwrap_or`s, but maybe extract into a constant?